app/views/components/filter-layout/index.html

```diff
@@ -0,0 +1,355 @@
+{% extends "layouts/base.html" %}
+
+{% block body %}
+
```

Comment:
To be consistent with other app examples, wrap the body content with the appropriate divs. For example:

```
{% block body %}
  <div class="govuk-grid-row">
    <div class="govuk-grid-column-full">
      <!-- backlink go here -->
      <!-- heading go here -->
      <!-- examples go here -->
    </div>
  </div>
{% endblock %}
```

Reply:
Why don't we just remove these extra divs from the other examples? They're not needed right.

Reply:
If we assume everything sits in a grid row and column, then they are needed. Either way, it would be easier adding them to this component, rather than removing them from numerous others. 

We can always reconcile the views at a later date to use the appropriate divs.